### PR TITLE
Update css for Desktop Project Selection 

### PR DESCRIPTION
**Pull request by maggiemah**
Original: 

Edit 1:

Edit 2:

Edit 3:

### CODE REVIEW

File: packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectProject.scss [hunk 1]
```diff
@@ -18,17 +18,35 @@
...
       }
 
       .iui-input-container {
-        grid-template: "label inputs icon" / auto minmax(50px, 290px) auto;
+        display: flex;
+        flex-direction: row;
+        width: 100%;
+        > .iui-label {
+          display: flex;
+          margin-left: 10px;
```
[the remark this answers is not in the document]
Comment (maggiemah) [reply packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectProject.scss:35]: I can use margin shorthand syntax,

**Comment by aruniverse** [comment]
Take a look at: https://itwindemo.bentley.com/view
I think its supposed to look like this, without the create project button

Also, make sure you update the same file in the template/desktop-viewer

**Comment by ben-polinsky** [comment]
Looks solid, @aruniverse not sure if you want the search on the right, in which case should be a simple fix.

**Comment by aruniverse** [comment]
> Looks solid, @aruniverse not sure if you want the search on the right, in which case should be a simple fix.

Yea lets, do the search on the right

@maggiemah

File: packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectProject.scss [hunk 2]
```diff
@@ -18,20 +18,72 @@
...
         min-width: 300px;
         padding-right: 61px;
       }
 
       .iui-input-container {
-        grid-template: "label inputs icon" / auto minmax(50px, 290px) auto;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        font-family: "Open Sans",-apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Helvetica Neue",Arial,sans-serif;
```
Comment (aruniverse) [inline packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectProject.scss:33]: wait, do you really need to set all this extra stuff? 
is this diverging too much from the design standards made by the components?

Comment (maggiemah) [reply packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectProject.scss]: Good point, defining the family isn't necessary but explicitly inheriting font-family and font-size is needed to change iui-input and iui-tab.

Comment (aruniverse) [reply packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectProject.scss]: any reason the font family or size needs to be changed at all?

Comment (maggiemah) [reply packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectProject.scss]: Seems to match better

font-family + font-size Inherited: 
image

Without: 
image

Comment (aruniverse) [reply packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectProject.scss]: ah dont worry too much about the exact look, the link i pointed too is using like a 2 yr old itiwnui ver
i wouldnt try to override the look too much, defeats the purpose of having our own custom library / design system

**Pull request merged**